linode/vpc_ips/datasource_test.go

```diff
@@ -0,0 +1,71 @@
+//go:build integration || vpc_ips
+
+package vpc_ips_test
```

Comment:
nitpick: Could this module be named `vpcips` to align with the naming of the other modules?

Reply:
Good call. Fixed!